docs/immortal-empires/vampire-counts.md

```diff
@@ -0,0 +1,33 @@
+# Vampire Counts
+
+## Mannfred von Carstein: The Drakenhof Conclave
+
+* **Route I - The Restless Dunes**
+  * The shifting sands of Nehekhara hide many long lost tombs filled with powerful magical artifacts that Mannfred desires. They also hide many foes who would see him undone. They will find only death and an unlife of eternal servitude in these dunes.
+  * Destroy factions:
+    * Khemri, Cult of Sigmar, Chevaliers de Lyonesse
+  * Control provinces:
+    * Shifting Sands, Land of the Dervishes, The Cracked Land, Land of the Dead, Coast of Araby, Land of Assassins, Great Desert of Araby
+  * **Reward**:
+    * _The Ossified Portal_: Creates a dilemma
+    * _The Ossified Monolith_: Campaign movement range: +10% for all armies, Diplomatic relations: +20 with Vampire Counts
```

Comment:
And this one feels a bit lackluster. Maybe +25% range? What do you think @Mjb141?

Reply:
Changed to 25% (be mindful I believe movement range actually caps at +50% total)

Found two really cool effects I added:
The Dead Rise Again: 10% chance for destroyed units to be restored with 10% of their total hit points.